http/headers/Age.json

```
@@ -7,14 +7,14 @@
           "spec_url": "https://www.rfc-editor.org/rfc/rfc9111#field.age",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "≤54"
```

Since `Age` is in https://www.rfc-editor.org/rfc/rfc2616.html#section-14.6 from 1999 I think it's relatively safe to just assume it's been supported since version 1:

```suggestion
              "version_added": "1"
```

---

Defered to https://github.com/mdn/browser-compat-data/pull/23719!